client/dist/index.html

```diff
@@ -3,9 +3,20 @@
 
 <head>
   <title>Template</title>
+  <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
+  <!-- Static Skeleton -->
+  <div class='container'>
+    <div class='top'>
+      <div id='musicPlayer'>music player</div>
```

Comment:
I've heard mixed opinions on the use of 'id' vs 'class' as identifiers. Check with the other team members regarding their preferred usage too so we can establish a standard.